apps/vs-code-designer/src/package.json

```diff
@@ -606,10 +601,6 @@
           "command": "azureLogicAppsStandard.openDesigner",
           "when": "resourceFilename==workflow.json"
         },
-        {
```

Comment:
You are removing this because you don't have the latest from main

Reply:
Merged with main